playbooks/azure/openshift-cluster/group_vars/all/yum_repos.yml

```diff
@@ -47,12 +47,13 @@ azure_node_repos:
     sslclientkey: /var/lib/yum/client-key.pem
     enabled: yes
 
-  #- name: rhel-server-7-ose-3.10
-  #  baseurl: https://mirror.openshift.com/libra/rhui-rhel-server-7-ose-3.10/
-  #  gpgkey: file:///etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release
-  #  sslclientcert: /var/lib/yum/client-cert.pem
-  #  sslclientkey: /var/lib/yum/client-key.pem
-  #  enabled: yes
+  - name: rhel-server-7-ose-3.10
+    baseurl: https://mirror.openshift.com/enterprise/all/3.10/latest/x86_64/os
```

Comment:
add a comment that this should change to (probably) `https://mirror.openshift.com/libra/rhui-rhel-server-7-ose-3.10/` post-GA

Reply:
:+1: